lib/galaxy/dependencies/conditional-requirements.txt

```diff
@@ -1,6 +1,6 @@
 # These dependencies are only required when certain config options are set
 psycopg2==2.6.1
-WebError==0.11
+WebError==0.10.3
```

Comment:
It was added here in commit a44b2c2170c9c326a2d5e1fec35beb44fd9834df , the in requirements.txt in 7c9c0cecdc3c31b9e86356470611ad1c7c0e87f4 , ping @natefoo.